src/include/miopen/mlo_internal.hpp

```diff
@@ -617,7 +617,7 @@ struct mlo_construct_pooling2D : mlo_construct_activ_lrn_pooling_common
         _pooling_method = MLO_POOLING_OP_MAX;
         _index_type     = miopenIndexUint8;
         _wsp_index      = miopenPoolingWorkspaceIndexMask;
-        _NAN_option     = 0;
```

Comment:
mmm ... this will change the naming convention (not saying it's a good one though IMHO) for others to keep track of it together with the (three) ones above.
@pfultz2 what would you recommend for the name of this member variable please? This is due to https://github.com/ROCmSoftwarePlatform/MIOpen/issues/1322

Reply:
`_nan_option`

Reply:
Replaced `NAN_option` with `_nan_option`. Thanks.